modules/cluster-management/src/main/java/org/apache/ignite/internal/cluster/management/ClusterInitializer.java

```diff
@@ -235,4 +262,20 @@ private static List<ClusterNode> resolveNodes(ClusterService clusterService, Col
                 })
                 .collect(Collectors.toList());
     }
+
+
+    private CompletableFuture<String> setDefaultsToClusterConfiguration(@Nullable String hocon) {
```

Comment:
Does it really have to be a future? I don't think that there are any asynchronous operations inside of the setter.